package.json

```diff
@@ -1,18 +1,25 @@
 {
   "name": "bitstyles",
-  "version": "1.0.0",
+  "version": "0.7.4",
   "description": "The Bitcrowd CSS library",
   "main": "bitstyles.scss",
   "private": true,
   "scripts": {
-    "build": "gulp build",
-    "lint": "gulp lint:scss lint:js",
-    "lint-js": "gulp lint:js",
-    "lint-scss": "gulp lint:scss",
-    "styleguide": "gulp styleguide",
-    "test:build": "gulp test:build",
-    "test:run": "gulp test:run",
-    "watch": "gulp watch"
+    "build": "node-sass bitstyles/bitstyles.scss -o build --output-style compact",
+    "postbuild": "postcss -c postcss_build.json",
+    "lint:scss": "postcss -c postcss_lint.json",
+    "lint:js": "eslint .",
+    "lint": "npm run lint:scss && npm run lint:js",
+    "test:build": "backstop reference",
+    "test": "backstop test",
+    "prestyleguide": "npm run build && cat build/bitstyles.css styleguide/assets/stylesheets/styleguide-extras.css > build/bitstyles_styleguide.css",
+    "styleguide": "styleguide --kss-source 'bitstyles/**/*.scss' --style-source 'build/bitstyles_styleguide.css' --output 'styleguide' --overviewPath 'README.md' --readOnly --watch --server --disableEncapsulation --title 'bitstyles' --extraHead '<script src=\"/assets/javascripts/a11y-dialog.js\"></script>''<script src=\"/assets/javascripts/styleguide-init.js\"></script>''<script src=\"/assets/javascripts/modal.js\"></script>''<script src=\"/assets/javascripts/expander.js\"></script>'",
+    "watch": "npm run build && npm run build -- --watch --recursive",
+    "clean": "rm -r build/*",
+    "precss:stats": "npm run build",
+    "css:stats": "parker build/bitstyles.css --format=json > stats/css.json",
+    "precss:graph": "npm run build",
+    "css:graph": "specificity-graph build/bitstyles.css --output stats"
```

Comment:
A task that runs both `lint` and `test` might be useful, e.g.

```json
"scripts": {
  "checks": "npm run lint && npm run test"
}
```

So that you can then…

```shell
npm run checks
```